base/db/R/dbfiles.R

```diff
@@ -251,13 +251,13 @@ dbfile.input.check <- function(siteid, startdate = NULL, enddate = NULL, mimetyp
   formatid <- get.id(table = "formats", colnames = c("mimetype_id", "name"), values = c(mimetypeid, formatname), con = con)
 
   if (is.null(formatid)) {
-    invisible(data.frame())
+    (invisible(data.frame()))
```

Comment:
Sorry, my previous comment was misreading the code, and I missed that you were making it return early on purpose. Yes, please _do_ add `return` here.

Reply:
Adding it now !! :)

Reply:
I think `invisible` prevents the automatic printing of data rather than stopping execution of the external called function all together.

Would it be better to feasible early `return` statements in more such instance wherever we will want to urgently return early and not continue?

Reply:
Yep, exactly right. Anywhere you see an `invisible(...)` that's not assigned to anything there's a great chance it was intended to stop execution and should be wrapped to `return(invisible(...))`. Always worth considering if an early return is still the right behavior, though.
(and at much lower priority, it's also worth considering whether the return should really be invisible or not, but in most cases the only difference that makes is in how verbose the log output gets)